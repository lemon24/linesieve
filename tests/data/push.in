--section '(\S+):$'
push t
  sub 1 2
  push r
    sub '(\d)' '\1\1'
---
one:
1
two:
1
three:
1
four:
1
