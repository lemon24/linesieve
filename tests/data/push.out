one
1
two
22
three
22
four
11
